package.json

```diff
@@ -65,7 +65,7 @@
         "@monaco-editor/react": "^4.1.3",
         "@popperjs/core": "^2.9.2",
         "@posthog/chart.js": "^2.9.6",
-        "@posthog/plugin-scaffold": "0.12.10",
+        "@posthog/plugin-scaffold": "^1.0.0-alpha.0",
```

Comment:
nit/q: I get that this is the schema going forward but should we get in an "alpha" version tag?

Reply:
Not sure what you mean - there's a TODO under this PR to bump scaffold before merging this and then updating this.